### ISSUE TITLE
Very low starting abundance error

### ISSUE THREAD

**Issue by isaacovercast** [isaacovercast/PIED#11]
Using these params out of 50 sims I get 3 errors converting nan to num:
```
------- PIED params file (v.0.0.2)----------------------------------------------
clads-horserace      ## [0] [simulation_name]: The name of this simulation scenario
./default_PIED       ## [1] [project_dir]: Where to save files
0.3                  ## [2] [birth_rate]: Speciation rate
taxa                 ## [3] [stop_criterion]: Whether to stop on ntaxa or time
50                   ## [4] [ntaxa]: Number of taxa to simulate if stop is `ntaxa`
4                    ## [5] [time]: Amount of time to simulate if stop is `time`
abundance            ## [6] [process]: Whether to evolve `abundance` or growth `rate` via BM
True                 ## [7] [ClaDS]: Whether to allow speciation rates to change along the branches a la ClaDS
50000                ## [8] [abundance_mean]: Ancestral abundance at time 0
0.0                  ## [9] [abundance_sigma]: Rate at which abundance changes if process is `abundance`
0.005                ## [10] [growth_rate_mean]: Ancestral population growth rate at time 0.
0.005                ## [11] [growth_rate_sigma]: Rate at which growth rate changes if process is `rate`
0.2                  ## [12] [ClaDS_sigma]: Rate at which speciation rate changes if ClaDS is True.
0.7                  ## [13] [ClaDS_alpha]: Rate shift if ClaDS is True
1000                 ## [14] [sequence_length]: Length of the genomic region simulated, in base pairs.
1e-08                ## [15] [mutation_rate]: Mutation rate per base per generation
5                    ## [16] [sample_size]: Number of samples to draw for calculating genetic diversity
```

**Comment by isaacovercast** [comment]
Fixed 0dc2ab2

**Comment by isaacovercast** [comment]
abundance_sigma must be > 0
